compiler+runtime/test/bash/error-reporting/src/analyze/invalid-throw/input.jank

```diff
@@ -0,0 +1,4 @@
+(ns input)
+
+(defn doworkson [a b]
+  (throw a b 3))
```

Comment:
GitHub thinks that a file named `output` is a binary format, so it's difficult to review. I can't even put this comment there :)

Reply:
Hmm, `output.txt` is still displays the same. Maybe it's a git thing? Couple of things to try:

https://stackoverflow.com/a/41240899
- `git rm` + `git add` or squash

https://stackoverflow.com/a/53287723
- see if git thinks it's binary `git check-attr --all -- src/my_file.txt`

Reply:
We'll see what happens after this is squashed and merged.